vernac/metasyntax.ml

```diff
@@ -947,6 +948,10 @@ let default = {
 
 end
 
+let warn_deprecated_ident_entry =
+  CWarnings.create ~name:"deprecated-ident-entry" ~category:"deprecated"
+         (fun () -> strbrk "grammar entry \"ident\" was misnomed in constr notations as it included \"_\" in addition to proper identifiers; it is deprectated; use \"name\" instead.")
```

Comment:
Then, maybe this is the message that could be enhanced to talk about the future:

```suggestion
         (fun () -> strbrk "grammar entry \"ident\" was misnomed in constr notations as it included \"_\" in addition to proper identifiers; it is deprectated and its meaning will change in the future; use \"name\" instead.")
```

Reply:
Yes.